examples/nlp/language_modeling/conf/megatron_t5_config_finetune_glue_eval.yaml

```diff
@@ -0,0 +1,34 @@
+name: megatron_t5_glue_eval
+
+trainer:
+  gpus: 8
```

Comment:
I think we may want to have default as just 1 GPU?

Reply:
cool